package.json

```diff
@@ -15,11 +15,15 @@
     "argon2": "^0.29.1",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
+    "from": "^0.1.7",
     "jsonwebtoken": "^8.5.1",
     "jwt-decode": "^3.1.2",
+    "lodash": "^4.17.21",
+    "lodash.shuffle": "^4.2.0",
     "pg": "^8.8.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "shuffle": "^0.2.5",
```

Comment:
Please take a moment to `npm uninstall` whichever packages you didn't end up using.